controllers/openstackdataplanenode_controller.go

```diff
@@ -241,7 +249,7 @@ func (r *OpenStackDataPlaneNodeReconciler) GenerateInventory(ctx context.Context
 	host.Vars["ansible_host"] = hostName
 
 	ansibleVarsData := make(map[string]interface{})
-	err = yaml.Unmarshal([]byte(instance.Spec.Node.AnsibleVars), ansibleVarsData)
+	err = yaml.Unmarshal([]byte(r.GetAnsibleVars(instance, instanceRole)), ansibleVarsData)
```

Comment:
@slagle here's the Getter you suggested.

Reply:
Great, my test looks like it's working now as well. :+1: